content/applications/productivity/knowledge/articles_editing.rst

```diff
@@ -0,0 +1,265 @@
+         * - :guilabel:`Switch direction`
+           - Switch the text's direction.
+         * - :guilabel:`Text`
+           - Paragraph block.
+
+   .. tab:: Media
+
+      .. list-table::
+         :widths: 20 80
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Command
+           - Use
+         * - :guilabel:`Image`
+           - Insert an image.
+         * - :guilabel:`Article`
+           - Link an article.
+
+   .. tab:: Navigation
+
+      .. list-table::
+         :widths: 20 80
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Command
+           - Use
+         * - :guilabel:`Link`
+           - Add a link.
+         * - :guilabel:`Button`
+           - Add a button.
+         * - :guilabel:`Appointment`
+           - Add a specific appointment.
+         * - :guilabel:`Calendar`
+           - Schedule an appointment.
+
+   .. tab:: Widget
+
+      .. list-table::
+         :widths: 20 80
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Command
+           - Use
+         * - :guilabel:`3 Stars`
+           - Insert a rating over 3 stars.
+         * - :guilabel:`5 Stars`
+           - Insert a rating over 5 stars.
+
+   .. tab:: Knowledge
+
+      .. list-table::
+         :widths: 20 80
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Command
+           - Use
+         * - :guilabel:`Table of Content`
+           - Add a table of content with the article's headings.
+         * - :guilabel:`Index`
+           - Show the first level of nested articles.
+         * - :guilabel:`Outline`
+           - Show all nested articles.
+         * - :guilabel:`Item Kanban`
+           - Insert a kanban view of article items.
+         * - :guilabel:`Item List`
+           - Insert a list view of article items.
+         * - :guilabel:`File`
+           - Embed a file that can be downloaded.
+         * - :guilabel:`Template`
+           - Add a template section that can be inserted in messages, terms & conditions, or
+             description in other applications.
+
+   .. tab:: Basic Blocks
+
+      .. list-table::
+         :widths: 20 80
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Command
+           - Use
+         * - :guilabel:`Signature`
+           - Insert your signature.
+
+Content from other app
+======================
+
+Knowledge allows to retrieve content views from other applications. To do so, go to the targeted app
+and create the desired view. Then, click :menuselection:`Favorite --> Insert view in article` and
+select an article. The view is inserted at the bottom of the selected article.
+
+.. example::
+   To retrieve the view below, we created it by going to :menuselection:`Sales --> Graph icon -->
+   Pie Chart icon` and inserted it by clicking :menuselection:`Favorite --> Insert view in article`
+   and selecting the *Sales Playbook* article.
+
+   .. image:: articles_editing/view-pie.png
```

Comment:
in this the focus is on the chart. It isn't very obvious that it is inserted inside an article.
I suggest taking a more contextual screenshot. Having a larger image would also reduce the displayed height (here it looks too big)

Reply:
what do you think about this one?